### PR TITLE
Dyn 4533 selection button style

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Adding and Applying the standard styling to the Dynamo Core Selection Button

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang

### FYIs

@RobertGlobant20 @filipeotero

### CODE REVIEW

**Comment by QilongTang** [comment]
hi @jesusalvino Thanks for the changes, did you figure out how to place selection node in DynamoSandbox and would you attach a UI difference screenshot as well?

File: src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml [hunk 1]
```diff
@@ -7,7 +7,75 @@
              mc:Ignorable="d" 
              d:DesignHeight="100" d:DesignWidth="200">
     <UserControl.Resources>
-        <converters:SelectionButtonContentConverter x:Key="SelectionButtonContentConverter" />
+        <ResourceDictionary>
```
Comment (mjkkirschner) [inline src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml:10]: @jesusalvino @QilongTang please see this comment I made on Long's recent PR - https://github.com/DynamoDS/Dynamo/pull/11958#discussion_r797938320

TLDR - I think this is an anti pattern and we should move the resource to the shared resource dictionary, not create it inside the control's dictionary or resources element.

Comment (mjkkirschner) [reply src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml:10]: the relevant quote is:

> A ResourceDictionary defined inside of a [UserControl](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.usercontrol) carries a penalty. The platform creates a copy of such a ResourceDictionary for every instance of the UserControl. If you have a UserControl that is used a lot, then move the ResourceDictionary out of the UserControl and put it at the page level.

File: src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml [hunk 2]
```diff
@@ -7,7 +7,75 @@
...
+                                        <SolidColorBrush Color="#FFFFFF" Opacity="0.5"/>
+                                    </Border.BorderBrush>
+
+                                    <TextBlock x:Name="textBlock"
+                                       HorizontalAlignment="Center"
+                                       Margin="0,1,0,0"
+                                       VerticalAlignment="Center"
+                                       FontFamily="{StaticResource ArtifaktElementRegular}"
+                                       FontSize="14px"
+                                       Foreground="#FFFFFF"
```
Comment (mjkkirschner) [inline src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml:55]: for all the colors defined here, it would be better to use the colors that are already defined in the dynamo modern and other resource dictionaries so that changing the color in the single place will update all controls, for example if we want to make themeing easy, or support different color schemes for accessibility.

File: src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml (hunk 1 above)
Comment (QilongTang) [reply src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml:10]: I agree, converter could stay but the style can be put in https://github.com/DynamoDS/Dynamo/blob/master/src/Notifications/Styles/DynamoModern.xaml

File: src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml (hunk 2 above)
Comment (jesusalvino) [reply src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml]: Done, thanks for the advice.

File: src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml (hunk 1 above)
Comment (jesusalvino) [reply src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml]: Done, thanks for the advice.

**Comment by jesusalvino** [comment]
> hi @jesusalvino Thanks for the changes, did you figure out how to place selection node in DynamoSandbox and would you attach a UI difference screenshot as well?

Per the team's answer in the channel there is not a way to do that, We were researching a bit about that but it look like need more time. the expected behavior has been attached.

imagen


SelectNode

**Comment by mjkkirschner** [comment]
It looks good to me - but I'm not really sure what it's supposed to look like :) (ie why is there an OK button?) did you apply this style to some dialog just for testing in sandbox?

Would be good to verify how it looks in Revit, could be done after merge.
Also please - quickly go through the checkboxes in the PR template if applicable.

Thanks for addressing the comments. 👍

**Review by mjkkirschner — APPROVED** [review]

**Comment by jesusalvino** [comment]
> 

ohh, the OK is just for comparison purpose, the expected behavior is the [Select] one, yes it was applied only for testing in Sandbox.
